client/src/lib/components/ui/Accordian/Accordian.svelte

```diff
@@ -0,0 +1,88 @@
+<script type="ts">
+    import downArrow from '$lib/assets/angle-down-solid.svg';
+    import type { IAccordian } from './Accordian.types';
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface $$Props extends IAccordian{};
+    export let title: $$Props['title'];
+    export let desc: $$Props['desc'];
+
+    let acc: HTMLElement;
+
+    const expand = () => {
+        acc.classList.toggle("activeAccordian");
+        const panel = acc.nextElementSibling as HTMLElement;
+        if (!panel) return;
+        if (panel.style.maxHeight !== '0px') {
+            panel.style.maxHeight = '0px';
+        } else {
+            panel.style.maxHeight = `${panel.scrollHeight}px`;
+        }
+    }
+</script>
+
+<div class="container">
+    <div class="accordian" bind:this={acc} on:keypress={expand} on:click={expand}>
+        <div class="title">
+            {title}
+        </div>
+        <div class="arrow">
+            <img src={downArrow} alt="Expand">
+        </div>
+    </div>
+    <div class="panel" style="max-height: 0px;">
+        <p>
+            {desc}
+        </p>
+    </div>
+</div>
+
+<style lang="scss">
+    :global(.activeAccordian) {
```

Comment:
global is not required here really.

just breaks our consistency with scss modules, try to remove this

Reply:
Svelte removes this from the styles as unused selectors. So, this is the only solution as far as I know.